tests/functional/oauth_webchannel.js

```diff
@@ -39,13 +39,13 @@ define([
    * finish OAuth flows
    */
 
-  function testIsBrowserNotifiedOfLogin(context, shouldCloseTab, wantKeys) {
+  function testIsBrowserNotifiedOfLogin(context, options) {
     return FunctionalHelpers.testIsBrowserNotified(context, 'oauth_complete', function (data) {
```

Comment:
We have to make sure `options` is defined before accessing one of its properties. An easy way to do this is to insert a line at line 43:

```js
options = options || {};
```
That ensures options is always defined and we can safely access its members.